app/models/form/sales/pages/person_known.rb

```diff
@@ -0,0 +1,22 @@
+class Form::Sales::Pages::PersonKnown < Form::Sales::Pages::Person
+  def initialize(id, hsh, subsection, person_index:)
+    super
+    @header_partial = "#{id}_page"
+    @header = ""
+    @description = ""
+    @subsection = subsection
+    @depends_on = page_depends_on
+  end
+
+  def questions
+    @questions ||= [
+      Form::Sales::Questions::PersonKnown.new(field_for_person("details_known_"), nil, self, person_index: @person_index),
+    ]
+  end
+
+  def page_depends_on
+    return (@person_index..4).map { |index| { "hholdcount" => index } } if @person_index == 1
```

Comment:
should this be for `person_display_number` not `@person_index`?

Reply:
Hm, I think person_index. Cause the index starts at 1 (and then 2, 3, 4). So it is the same as household count.
I don't think we use the `person_display_number` for `details_known` fields at all, because they would always be from 1 to 4, no matter if it's a joint purchase or not.